python/scripts/analytics_api_example.py

```diff
@@ -41,7 +41,9 @@ def main(argv=[]):
     # Get configuration from defaults and/or the environment.
     # Set the API configuration verbosity to 2 to show all of requests
     # and response statuses. To see the complete responses, set verbosity to 3.
-    api_config = ApiConfig(verbosity=args.log_level, version=args.api_version)
+
+    # Set API version to 3, because this script does not work with 5 yet.
+    api_config = ApiConfig(verbosity=args.log_level, version='3')
```

Comment:
Should this be `version='v3'`? It looks like the `v` prefix is used elsewhere.

Reply:
Good question! Version is specified as v3 or v5 everywhere except for the arguments. From a user's perspective, it's redundant to type '-v v3' or '-v v5' so the command line arguments are shortened to '-v 3' and '-v 5'. Based on developer feedback, I might switch to accept v3 and v5 in addition to the 3 and 5 shortcuts.